features/org.wso2.carbon.appmgt.core.feature/src/main/resources/jaggery-modules/store/scripts/server.js

```diff
@@ -131,8 +148,11 @@ var server = {};
                 secured: false
             };
         }
-        //loads the tenant if it hasn't been loaded
-        server.loadTenant(obj);
+
+        if(obj.tenantId != -1){
```

Comment:
Fixed in https://github.com/wso2/carbon-appmgt/pull/539/commits/207d3b1c14192fee91f6d02b39c54f42bdf46e92